app/org/sagebionetworks/bridge/models/studies/StudyParticipant.java

```diff
@@ -0,0 +1,81 @@
+package org.sagebionetworks.bridge.models.studies;
+
+public final class StudyParticipant {
```

Comment:
Subclassable? No, and makes the equals/hashCode methods work correctly without changing the default implementation provided by Eclipse. Fields aren't final simply so I can use setters rather than collecting all the data for a large set of constructor arguments. Okay for instances to be mutable.